include/fluent-bit/flb_s3_local_buffer.h

```diff
@@ -70,5 +70,7 @@ int flb_remove_chunk_files(struct flb_local_chunk *c);
 
 void flb_chunk_destroy(struct flb_local_chunk *c);
 
+void flb_local_buffer_destroy(struct flb_local_buffer *store);
+
```

Comment:
Did you do `git rebase` with the base branch? My new changes shouldn't show up in your PR